src/app/helpers/announcement/controller.js

```diff
@@ -396,19 +398,18 @@ class AnnouncementController {
         table: this.objectStoreRest.MODEL.ATTACHMENT,
         values: {
           'id': attachmentId,
-          'file': requestObj.file.buffer.toString('utf8'),
+          'file': requestObj.file.buffer.toString('base64'),
           'filename': requestObj.file.originalname,
           'mimetype': requestObj.file.mimetype,
           'status': 'created',
-          'createddate': dateFormat(new Date(), "yyyy-mm-dd HH:MM:ss:lo")
+          'createddate': dateFormat(new Date(), "yyyy-mm-dd HH:MM:ss:lo"),
         }
       }
-
       if (!_.isEmpty(requestObj.body.createdBy)) query.values.createdby = requestObj.body.createdBy
 
       try {
         return await (new Promise((resolve, reject) => {
-          this.objectStoreRest.createObject(query)
+          this.objectStoreRest.createObject(query, false)
```

Comment:
Don't pass boolean directly. Should create a variable and pass that. This helps with maintainability.